setup.py

```diff
@@ -17,7 +17,7 @@ def _read_reqs(relpath):
 
 setuptools.setup(
     name='cloudflare-exporter',
-    version='0.6',
+    version='0.7',
```

Comment:
Split in separate commit

Reply:
discuss offline, we will use https://docs.openstack.org/pbr/2.1.0/ for next release